test/node-rpc-test.js

```diff
@@ -177,6 +177,28 @@ describe('RPC', function() {
       const connectionsCnt = await nclient.execute('getconnectioncount', []);
       assert.strictEqual(connectionsCnt, 1);
     });
+
+    it('should rpc setban a peer',async () => {
+      let listbanned;
+      let connectionsCnt;
+      // getting initial connection count
+      connectionsCnt = await nclient.execute('getconnectioncount', []);
+      assert.strictEqual(connectionsCnt, 1);
+      // getting initial banned list
+      listbanned = await nclient.execute('listbanned', []);
+      assert.strictEqual(listbanned.length, 0);
```

Comment:
Mine as well compact this:

```suggestion
      // getting initial connection count
      let connectionsCnt = await nclient.execute('getconnectioncount', []);
      assert.strictEqual(connectionsCnt, 1);
      // getting initial banned list
      let listbanned = await nclient.execute('listbanned', []);
      assert.strictEqual(listbanned.length, 0);
```